docs/install/any-kubernetes-cluster.md

```diff
@@ -551,7 +551,7 @@ The following command installs an implementation of Channel that runs in-memory.
 The following command installs an implementation of Broker that utilizes Channels:
 
    ```bash
-   kubectl apply --filename {{< artifact repo="eventing" file="channel-broker.yaml" >}}
+   kubectl apply --filename {{< artifact repo="eventing" file="deprecated-channel-broker.yaml" >}}
```

Comment:
I think it might be better to actually just point to mt-channel-broker so the folks don't have to migrate away from it? WDYT?
If you do decide to do that, you also need to change L570 to be:
brokerClass: MTChannelBasedBroker

Reply:
@vaikas Done